package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jupiterone/eslint-config",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
Given this is kind of a new feature of the linting config, I wonder if we should consider making this a minor version bump. It's also a potentially breaking change for some devs so a major version bump could be considered (though in this case, it's a breaking change we want to force users to adopt).